server/project/api/events_handler.py

```diff
@@ -9,6 +9,17 @@
 from project.error_handlers import *
 import datetime as dt
 
+
+def verify_at_least_1_day_available(availability: Availability) -> int:
+    """Counts the number of days selected as available and returns the number
+    as an integer."""
+    days_available = 0
+    for day in availability['days'].values():
+        if day:
+            days_available += 1
+    return days_available
+
```

Comment:
Could also do 

`return any(availability['days`].values()`

Reply:
Implemented thank you!